fellow-submissions/peter-rose/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+  <div class="container-wrapper">
+    <main class="container">
+      <header>
+        <h4>Quiz'n You, my guy</h4>
+      </header>
+      
+      <section class="q1">
+      <h4>Info Please</h4>
+      <div style="float:left">
+    <label for="name">Name</label>
+    <input id="name" type="text" value="" name="name">
+  </div>
+
+  <div style="float:left;">
```

Comment:
Gotta get rid of this. 
1. No reason to have inline styling here.
2. **No use of floats for layout.**